dvc/tree/azure.py

```diff
@@ -170,19 +170,24 @@ def getsize(self, path_info):
     def get_file_hash(self, path_info):
         return HashInfo(self.PARAM_CHECKSUM, self.get_etag(path_info))
 
-    def _upload(
-        self, from_file, to_info, name=None, no_progress_bar=False, **_kwargs
-    ):
-
+    def _upload_fobj(self, fobj, to_info):
         blob_client = self.blob_service.get_blob_client(
             to_info.bucket, to_info.path
         )
+        blob_client.upload_blob(fobj, overwrite=True, max_concurrency=1)
```

Comment:
Is `max_concurency=1` needed? If so, does it affect the upload speed? We didn't set it to 1 earlier.